tests/integration/thrift_tests.py

```diff
@@ -589,22 +590,20 @@ def is_healthy(self, context, req=None):
                 self.assertEqual(handler.probe, IsHealthyProbe.LIVENESS)
 
 
-class ThriftErrorReplacementTests(GeventPatchedTestCase):
```

Comment:
Is this test not accurate anymore?

Reply:
Ooops, I meant to delete the code that was added in this patch. Seems I missed and hit the preceding block 😓 Fixed.